src/main/scala/translating/BAPToIR.scala

```diff
@@ -62,23 +62,19 @@ class BAPToIR(var program: BAPProgram, mainAddress: Int) {
   private def translate(s: BAPStatement) = s match {
     case b: BAPMemAssign   => MemoryAssign(b.lhs.toIR, b.rhs.toIR, Some(b.line))
     case b: BAPLocalAssign => LocalAssign(b.lhs.toIR, b.rhs.toIR, Some(b.line))
-    case _                 => throw new Exception("unsupported statement: " + s)
   }
```

Comment:
I get a Pattern Match Exhaustivity Warning here now due to not handling `BapAssign`.